### ISSUE TITLE
Incorrect ERRORs reported when importing csv file - Duplicate entry for Primary keys

### ISSUE THREAD

**Issue by salvor-hardin** [Sequel-Ace/Sequel-Ace#173]
- Sequel Ace Version: 2.1.0
- macOS Version: 10.15.5
- MySQL Version: 5.7.30

**Description**
Incorrect ERRORs reported when importing csv file - Duplicate entry for Primary keys

**Steps To Reproduce**
Create this table:

CREATE TABLE `varchars` (
  `varchar` varchar(4) NOT NULL DEFAULT '',
  PRIMARY KEY (`varchar`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;

Then import the attached CSV file
[varchars.txt](https://github.com/Sequel-Ace/Sequel-Ace/files/4879759/varchars.txt)

varchars
0800
0700
0200
2000
0202
0202

With defaults:
<img width="647" alt="2020-07-06 at 16 07 45" src="https://user-images.githubusercontent.com/40695717/86609092-57183480-bfa3-11ea-99ca-1081a968a5d6.png">

Sequel Ace will report excessive primary key duplication, i.e. "0800" is there only once to be inserted.
<img width="744" alt="2020-07-06 at 16 08 02" src="https://user-images.githubusercontent.com/40695717/86609155-6e572200-bfa3-11ea-8f5c-1d0f519759f0.png">

**Expected Behaviour**
Only "0202" should be reported as a duplicate entry.

**Additional Context**
Same issue in Sequel Pro.

**Comment by Jason-Morcos** [comment]
@salvor-hardin I tried this myself and only got a duplicate key error for 0202. Are you certain you truncated the table/the table was empty before trying the import?

**Comment by salvor-hardin** [comment]
Hi:

I've just recreated the table and got the same errors when importing

Have you imported using the same settings I have in both screens?

<img width="652" alt="2020-07-07 at 10 36 55" src="https://user-images.githubusercontent.com/40695717/86761567-f68a0580-c03d-11ea-96b2-105150e22ac7.png">


<img width="733" alt="2020-07-07 at 10 37 13" src="https://user-images.githubusercontent.com/40695717/86761608-fd187d00-c03d-11ea-8ba1-1ee1275b9dfe.png">

.


(exact same issue with Sequel Pro)
Also, after the import Sequel Ace refreshes the number of rows to 5 but the size is kept @ 0, it needs a refresh to update the size.

**Comment by Jason-Morcos** [comment]
Fascinating, so I had created the varchars table myself. using my own create statement
> CREATE TABLE `varchars` (
>   `varchar` varchar(4) NOT NULL DEFAULT '',
>   PRIMARY KEY (`varchar`)
> ); 

<img width="1223" alt="Screen Shot 2020-07-07 at 18 23 05" src="https://user-images.githubusercontent.com/10710367/86862938-0d9d1780-c07f-11ea-8503-67d141e5d259.png">
<img width="193" alt="Screen Shot 2020-07-07 at 18 24 52" src="https://user-images.githubusercontent.com/10710367/86862991-29a0b900-c07f-11ea-8a92-f2826f51d22f.png">

This yields the proper error
<img width="393" alt="Screen Shot 2020-07-07 at 18 25 41" src="https://user-images.githubusercontent.com/10710367/86863136-6cfb2780-c07f-11ea-8800-2f71ae371a2c.png">


However, using your customized create statement
> CREATE TABLE `varchars` (
>   `varchar` varchar(4) NOT NULL DEFAULT '',
>   PRIMARY KEY (`varchar`)
> ) ENGINE=MyISAM DEFAULT CHARSET=latin1;

<img width="1225" alt="Screen Shot 2020-07-07 at 18 29 16" src="https://user-images.githubusercontent.com/10710367/86863313-cbc0a100-c07f-11ea-8531-a23591a854b3.png">
<img width="187" alt="Screen Shot 2020-07-07 at 18 27 58" src="https://user-images.githubusercontent.com/10710367/86863262-b8153a80-c07f-11ea-85aa-a61f572cc97a.png">



Yields the weird error
<img width="387" alt="Screen Shot 2020-07-07 at 18 27 42" src="https://user-images.githubusercontent.com/10710367/86863222-a0d64d00-c07f-11ea-998c-a331d7060700.png">


The only difference to me appears to be the engine.

Any ideas @jamesstout @gboudreau ?

**Comment by Jason-Morcos** [comment]
As we have received no reports of this issue in version 3+ of Sequel ace, I'm gonna close this issue. If you experience this issue again in the latest release of Sequel Ace, please create a new issue and reference this issue number in your description.
